BogoSort.scala

```diff
@@ -1,53 +1,37 @@
 import util.Random
 
-object BogoSort
-{
-	 def CheckIfSorted(args : Array[Int],  size:Int):Int=
-	{
-		for(i <- 0 to size-1)
-		{
-			if(args(i)>args(i+1))
-			return 0
-		}
-		return 1
-	}
+object BogoSort {
+  def CheckIfSorted(args: Array[Int], size: Int): Boolean = {
+    for (i <- 0 until size) {
+      if (args(i) > args(i + 1))
+        return false
+    }
+    true
+  }
 
-	def shuffle(args: Array[Int], size:Int)
-	{
-		var r:Int=0;
-		var t:Int=0;
-		for(i <- 0 to size)
-		{
-			// println(args(i))
-			t=args(i);
-			r=((Random.nextInt(100))%size)
-			// println(r)
-			args(i)=args(r);
-			args(r)=t;
-		}
-	}
+  def shuffle(args: Array[Int], size: Int): Unit = {
+    var r: Int = 0
+    var t: Int = 0
+    for (i <- 0 to size) {
+      t = args(i)
+      r = Random.nextInt(size)
```

Comment:
No need for `% size`. If a random number between `0` to `x` is needed, use `Random.nextInt(x)` and not `Random.nextInt(y) % x`